selfdrive/car/hyundai/values.py

```diff
@@ -1158,7 +1160,7 @@ class Buttons:
       b'\xf1\x82CNCWD0AMFCXCSFFA',
       b'\xf1\x82CNCWD0AMFCXCSFFB',
       b'\xf1\x82CNCVD0AMFCXCSFFB',
-      b'\xf1\x870\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf1\x82CNDWD0AMFCXCSG8A',
+      b'\xf1\x870\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf1\x81HM6M2_0a0_G80',
```

Comment:
I checked @jdtram's routes and he has this new FW version for engine. Thanks for updating this!

Reply:
I accidently added values to initial commit for entries that had brand not Hyundai. When I went back through deleting those that had non Hyundai brand I accidently deleted this entry @jdtram had entered so I wanted to let him know and possibly get things checked on his end in case it's need for his side.
Should I need to do anything else regarding functionality of my car with openpiliot now that I've updated the fingerprint for my vehicle's firmware?

Reply:
Nope you're good, his fingerprints matched what you added here.

Reply:
Sorry, I’m confused where it was said that I added VW info.

Reply:
Wahoo. Now I just wanna keep checking other cars to add their fingerprints =D Could I pull this data with just the odb power? Sorry off topic. I'll see what I can find tomorrow. Have a good night.

Reply:
Sure, go ahead. The wiki is open source. I'd add anything you think is not super obvious, and there's probably a lot to remove/simplify there

Reply:
> Sorry, I’m confused where it was said that I added VW info.

In the process of updating this my devices firmware I accidently added these entries to the file if they were missing
output simplified
`{'address': 2016,
 'brand': 'toyota',
 'ecu': 'engine',
 'fwVersion': b'HM6M2_0a0_G80',
 'request': [b'>\x00', b'\x10\x01', b'\x10\x03', b'"\xf1\x81'],
{'address': 2016,
 'brand': 'subaru',
 'ecu': 'engine',
 'fwVersion': b'CNDWD0AMFCXCSG8A',
{'address': 2017,
 'brand': 'subaru',
 'ecu': 'transmission',
 'fwVersion': b'CCN0N20NS5\x00\x00\x00\x00\x00\x00',
{'address': 2016,
 'brand': 'volkswagen',
 'ecu': 'engine',
 'fwVersion': b'\xf1\x870\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xf1\x82CNDWD0AMFCXCSG8A',
{'address': 2017,
 'brand': 'volkswagen',
 'ecu': 'transmission',
 'fwVersion': b'\xf1\x87CXNQF45856792JL2wvug\x99UcV\xaavffgVwyff_\xffy\xff\xff\xff\xf14\xf1\x89HT6VA640A1\xf1\x82CCN0N20NS5'
              b'\x00\x00\x00\x00\x00\x00',
`
After initial pull request discord let me know not to add these and so I delete them for second commit and then realized on the diff I had deleted a line I hadn't added. I went back on the commit to try and find who had added that line so I could let them know and I cam across you with https://github.com/commaai/openpilot/pull/23818

Reply:
Gotcha thanks.  Sorry. I misread what you’d said and thought I had severely messed up somewhere.